server/controllers/api/assessmentDayDataController/assessmentDayDataController.test.js

```diff
@@ -449,7 +446,16 @@ describe('assessmentDayDataController', () => {
       it('returns a status of 400 and an error message', async () => {
         const request = createRequest({
           body: {
-            metadata: { assessment: 'assessment' },
+            metadata: createAssessmentDayDataMetadata({
+              study: 'study',
+              participant: 'participant',
+              assessment: 'assessment',
+              units: 'day',
+              start: '1',
+              end: '4',
+              time_end: '4',
+              Consent: new Date('2020-01-02'),
+            }),
```

Comment:
not a big deal but we should look for opportunities to remove the attributes we don't care about for the tests to trim down the diff.

```suggestion
            metadata: createAssessmentDayDataMetadata(),
```